views/layout.etlua

```diff
@@ -1,36 +1,41 @@
 <!DOCTYPE html>
 <html>
-	<head>
-		<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <link rel="stylesheet" href="static/stylesheet.css">
-		<title><%= page_title or i18n("website_name") %></title>
-	</head>
-	<body>
+    <title><%= page_title or i18n("website_name") %></title>
+  </head>
+  <body>
     <header id="header">
       <img style='height: 100%; width: 10%; object-fit: contain; align=left' src="static/images/mudlet.png">
-		</header>
-		<div id="wrapper">
-			<main>
-				<div id="content">
-					<div class="innertube">
-          		<% if errors then %>
-		            <% render("views.error") %>
-		          <% end %>
-						  <% content_for("inner") %>
-					</div>
-				</div>
-			</main>
-			<nav id="nav">
-				<div class="innertube">
-					<h3><%= i18n("navigation_title") %></h3>
-					<ul>
-						<li><a href="<%= url_for("index") %>"><%= i18n("index_nav") %></a></li>
-						<li><a href="<%= url_for("login") %>"><%= i18n("login_nav") %></a></li>
-						<li><a href="<%= url_for("register") %>"><%= i18n("register_nav") %></a></li>
-						<li><a href="<%= url_for("packages") %>"><%= i18n("packages_nav") %></a></li>
-					</ul>
-				</div>
-			</nav>
-		</div>
-	</body>
-</html>
\ No newline at end of file
+    </header>
+    <div id="wrapper">
+      <main>
+        <div id="content">
+          <div class="innertube">
+          <% if errors then %>
+            <% render("views.error") %>
```

Comment:
This should probably only show for admin, not regular users? Showing them an error won't do much - it's not like they can do anything about it.

Reply:
I mean, it's nice to know when you registration fails, or your login doesn't work because of invalid credentials? I'd find it more frustrating to just keep putting them in and getting back the same page with no explanation as to what happened.

Reply:
This is to say, this is what shows the error messages we set in the i18n files, things like "Username and password don't match" , not stack trace.

Reply:
Ahh ok